sections/06_discussion.md

```diff
@@ -49,6 +49,86 @@ with only a couple GPUs.*
 *Some of this is also outlined in the Categorize section.  We can decide where
 it best fits.*
 
+Efficiently scaling deep learning is challenging, and there is a high
+computational cost (e.g., time, memory, energy) associated with training neural
+networks and using them for classification. For these reasons, neural networks
```

Comment:
Is it training them and using them or primarily training that is resource intensive?